### ISSUE TITLE
Clean install of 0.20.1 can't use network services

### ISSUE THREAD

**Issue by TheNavyBear** [Stellarium/stellarium#1105]
### Expected Behaviour
When opening Stellarium it should get my location from network. It should also be able to download star catalogs.

### Actual Behaviour
When opening a clean install of Stellarium 0.20.1 (with the 'get location from network' box checked), it says I am in Paris (not right). When I attempt to download extra star catalogs I get 'Error creating SSL context ( )'.

### Steps to reproduce
Setup a new PC or wipe an existing PC, load a fresh OS install, and do a fresh install of Stellarium 0.20.1

### System
* Stellarium version: Stellarium 0.20.1 (installed using Chocolatey)
* Operating system: Windows 10 Home 1909  -  Build 18363.836
* Graphics Card: Intel HD Graphics 615  -  Driver 26.20.100.7637
* Screen type (if applicable): Resolution 1920x1080, Scaling 150%

### Logfile 
[log.txt](https://github.com/Stellarium/stellarium/files/4685563/log.txt)

**Comment by TheNavyBear** [comment]
I will note that I have another PC that was setup with a fresh Chocolatey install of Stellarium 0.19.something that Chocolatey has updated to each new version, and is now also at 0.20.1 and it get location from network just fine. The extra star catalogs were downloaded when I first set it up, and now it says they are all installed and I don't have an option to download any to test.

**Comment by alex-w** [comment]
This is very strange. Could you check the latest beta as clean install?

**Comment by TheNavyBear** [comment]
I will attempt too if I get some time.

**Comment by alex-w** [comment]
I'm checked clean install of version 0.20.1 into Windows 7 (x64) and I cannot reproduce any issues for work of network services.

**Comment by TheNavyBear** [comment]
I am downloading the latest Beta build now to test. I have completely removed the old install including files in the user's AppData folder and will try a clean install of Beta. Sorry it took a while to get the time to test. Will post my results as soon as it downloads and installs.

**Comment by gzotti** [comment]
Have you allowed Stellarium to connect through the firewall?

**Comment by TheNavyBear** [comment]
A clean install of stellarium-0.20.1.17986-win64 works fine. I will try uninstalling it now and see if the release version works this time.

**Comment by TheNavyBear** [comment]
> Have you allowed Stellarium to connect through the firewall?

Yes, for purposes of testing before making the bug report, I had turned the firewall off with no luck. The release version doesn't work, but a clean install of stellarium-0.20.1.17986-win64 works fine, with no adjustments to the firewall.

**Comment by TheNavyBear** [comment]
So the Beta version, when clean installed, works fine. The release version, when clean installed, does not. Even if I install then remove the Beta version, then install the release version, it does not work.

The Beta version started right up to my current location from the network, and downloaded extra star files (I did the first two as a test). The release version starts up with the location of Paris, and trying to download extra star files results in the SSL error above. 

Also, as stated above, I have another Windows 10 machine that was setup in February with the then-latest Stellarium release, that then got updated with each new version. It runs 0.20.1 just fine that way, getting the location from network on each boot (again extra star files were downloaded with first install, so couldn't test those, but all other network stuff works).

Both of these machines run Windows 10 on Intel CPUs with Intel integrated graphics. Hopefully whatever fix is in the Beta version makes it's way to the next stable release.
